### PR TITLE
Updating hubspot language forms

### PR DESCRIPTION
**Pull request by JMStudiosJoe**

### CODE REVIEW

File: src/compositions/Video/Video.js [hunk 1]
```diff
@@ -31,6 +27,7 @@ const Video = ({ video }) => {
           onStateChange={ onStateChange }
         />
       </div>
+      <h4>You must watch the full video without stopping before taking the survey.</h4>
```
Comment (noelsner) [inline src/compositions/Video/Video.js:30]: Will this text change depending on the language?

File: src/compositions/HubspotForm/HubspotForm.css [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 .hubspot .titleText {
   color: #2F3772;
-}
+}
```
Comment (noelsner) [inline src/compositions/HubspotForm/HubspotForm.css:7]: Extra space after bracket

File: src/containers/MainContainer/MainContainer.js [hunk 3]
```diff
@@ -10,20 +10,43 @@ import './MainContainer.css';
...
   return (
     <div className='MainContainer'>
       <div className='wrapper'>
         <div className='items'>
           <LandingPage welcomeMessage={ content[language].welcomeMessage }/>
           <ChooseLanguage language={ content[language] } setLanguage={ setLanguage } />
-          <Video video={ content[language].video } />
-          <HubspotForm hubspot={ content[language].hubspot } />
+          <Video onEnd={ videoEndedHandler } video={ content[language].video } />
+          <HubspotForm hubspot={ content[language].hubspot } hasWatchedVideo={ hasWatchedVideo } language={ language } />
```
Comment (noelsner) [inline src/containers/MainContainer/MainContainer.js:49]: Looks great!

File: src/compositions/Video/Video.js (hunk 1 above)
Comment (JMStudiosJoe) [reply src/compositions/Video/Video.js:30]: not yet needs to though

**Pull request merged**